cmd/flagger/main.go

```diff
@@ -91,6 +92,7 @@ func init() {
 	flag.IntVar(&kubeconfigBurst, "kubeconfig-burst", 250, "Set Burst for kubeconfig.")
 	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&metricsServer, "metrics-server", "http://prometheus:9090", "Prometheus URL.")
+	flag.BoolVar(&insecureSkipVerify, "insecure-skip-verify", false, "disable verification of certificate for metrics server, this is insecure")
```

Comment:
Please remove this, `insecureSkipVerify` should be possible only via MetricTemplates, the global metric provider will go away soon.

Reply:
fixed!